src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+server.port=8005
+
+telegrambot.botUserName=
```

Comment:
Where is my `enviroment variable`?  And please rework the configuration in `YAML` format.

Reply:
Replaced application.properties with yaml.properties.
And set the environment variable